go.sum

```diff
@@ -63,6 +63,8 @@ github.com/BurntSushi/toml v1.1.0 h1:ksErzDEI1khOiGPgpwuI7x2ebx/uXQNw7xJpn9Eq1+I
 github.com/BurntSushi/toml v1.1.0/go.mod h1:CxXYINrC8qIiEnFrOxCa7Jy5BFHlXnUU2pbicEuybxQ=
 github.com/BurntSushi/xgb v0.0.0-20160522181843-27f122750802/go.mod h1:IVnqGOEym/WlBOVXweHU+Q+/VP0lqqI8lqeDx9IjBqo=
 github.com/DataDog/datadog-go v3.2.0+incompatible/go.mod h1:LButxg5PwREeZtORoXG3tL4fMGNddJ+vMq1mwgfaqoQ=
+github.com/DerAndereAndi/eebus v0.0.0-20220812150617-4f7442d9e071 h1:W0EymUrYE801OdntqBGkNGIMyi9AS1tsznnehCI07z4=
```

Comment:
Das ist schräg. Hier müsste

    github.com/evcc-io/eebus 

stehen da es ja ein PR gegen dieses Repo ist???

Reply:
Es ist ein PR und kein Branch in dem Repo. Also geht das erst wenn der PR dort gemerged ist.